z2/src/bin/z2.rs

```diff
@@ -444,9 +444,12 @@ struct DepositStruct {
     /// Specify the staking reward address
     #[clap(long, short)]
     reward_address: String,
+    /// Specify the signing address
+    #[clap(long, short)]
+    signing_address: String,
     /// Specify the Validator Proof-of-Possession
```

Comment:
shall `--deposit-auth-signature` still be called `Validator proof-of-possession`?

@DrZoltanFazekas @JamesHinshelwood ?

Reply:
let's rename it to BLS signature as @JamesHinshelwood suggested above